core/src/main/resources/lib/layout/task.jelly

```diff
@@ -136,6 +140,11 @@ THE SOFTWARE.
                 <l:icon src="${icon}" />
               </span>
               <span class="task-link-text">${title}</span>
+              <j:if test="${attrs.badge != null}">
+                <span class="task-icon-badge">
```

Comment:
This seems inconsistent with the naming convention you're generally using, why?

Reply:
I'm just following the existing naming convention for this component - I'd be happy to rename the classes (and a general cleanup of the component) but that'd be out of scope for this PR.